src/Tooling/Bit.Tooling.Templates/TodoTemplate/Api/Resources/EmailConfirmation.cshtml

```diff
@@ -9,23 +9,105 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Confirm email</title>
+    <style>
+
+        body {
+            margin: 0;
+            background-color: #FCFCFC;
+            font-family: 'Segoe UI';
+        }
+
+        .page-container {
+            display: flex;
+            flex-flow: column nowrap;
+            justify-content: space-evenly;
+            align-items: center;
+            height: 100vh;
+        }
+
+        .welcome-msg-box {
+            display: flex;
+            flex-flow: column nowrap;
+            justify-content: space-evenly;
+            align-items: center;
+            border: 1px solid #EDEBE9;
+            width: 83.33%;
+            height: 60%;
+        }
+
+        .welcome-msg-txt {
+            font-style: normal;
+            font-weight: 600;
+            font-size: 24px;
+            line-height: 36px;
+            color: #0D2960;
+        }
+
+        .welcome-msg-desc {
+            font-weight: 400;
+            font-size: 18px;
+            line-height: 28px;
+            text-align: center;
+            width: 65%;
+        }
+
+        .confirm-btn {
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            height: 38px;
+            width: 143px;
+            border-radius: 2px;
+            font-weight: 600;
+            font-size: 16px;
+            border: none;
+            background-color: #0078D4;
+            color: white;
+            text-decoration: unset;
+        }
+
+        .page-footer {
+            display: flex;
+            flex-flow: column nowrap;
+            justify-content: space-between;
+            align-items: center;
+            height: 50px;
+        }
+
+        .logo-lnk {
+            font-weight: 400;
+            font-size: 12px;
+            line-height: 18px;
+            color: #0065EF;
+            text-decoration: unset;
+        }
+
+        .logo-img {
+            width: 32px;
+            height: 21.33px;
+        }
+    </style>
 </head>
 <body>
-    Hello @Model.DisplayName
-    <br />
-    <br />
-    Thanks for using this app.
-    <br />
-    <br />
-    Please verify your email address.
-    <hr />
-    <a href="@Model.ConfirmationLink">Click to confirm!</a>
-    <br />
-    <br />
-    Or, copy the link below to your browser address bar:
-    <br />
-    <br />
-    @Model.ConfirmationLink
-    <hr />
+    <div class="page-container">
```

Comment:
Have you checked the output of the work on the email? 
The best way to create a template email is to use a table, not a div.

Reply:
Yes, I checked the output. It looks good...

<img width="557" alt="Untitled" src="https://user-images.githubusercontent.com/62190682/165635239-61d7f0ee-ac42-4635-a7dd-4518efe65485.png">

Do you think I need to use the table?

Reply:
Plz, check the output in Gmail, the outlook website, and the outlook application. smtp4dev is not trusted.